### PR TITLE
Feature/abr documentation

### PR DESCRIPTION
**Pull request by MarcoTodescato**
PR to implement first action workflow to build and deploy documentation to pages

### CODE REVIEW

**Comment by MarcoTodescato** [comment]
@patrickkesper here is the PR ... check whether you can participate in it :-)

**Comment by patrickkesper** [comment]
The PR looks good to me :+1:

File: .github/workflows/build-documentation.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Workflow to build and publish latest documentation from abr/feature/build-documentation
+
+name: build ABR documentation
+on: 
+  push:
+    branches:
+      - feature/abr-documentation
```
Comment (patrickkesper) [inline .github/workflows/build-documentation.yml]: I guess, this should be changed to master, right?

- [x] Add to master

File: .github/workflows/build-documentation.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        uses: actions/checkout@v2  # checkout code
+      
+      - name: Setup project
+        run: pip install -r requirements.txt  # install additional requirements (among which sphinx theme)
+      
+      - name: Build-Docs
+        run: mkdocs build
+      
+      - name: Deploy Pages
+        # env:
```
Comment (patrickkesper) [inline .github/workflows/build-documentation.yml]: This part can be removed since we do not need secrets here

- [x] Remove secrets

**Review by patrickkesper — CHANGES_REQUESTED** [review]

**Review by patrickkesper — APPROVED** [review]

File: docs/index.md [hunk 3]
```diff
@@ -32,99 +24,44 @@ The workflow of using ABR is held as simple as possible.
...
+```
 
-Contacts
-========
+## Contacts<a name="contacts"></a>
 
-| `Nicolai Waniek <mailto:Nicolai.Waniek@de.bosch.com>`_
-| `Marco Todescato <mailto:Marco.Todescato@de.bosch.com>`_
-| `Markus Spies <mailto:Markus.Spies2@de.bosch.com>`_
+- [Nicolai Waniek](mailto:Nicolai.Waniek@de.bosch.com)
```
Comment (MarcoTodescato) [inline docs/index.md]: @nwaniek what do you want us to do with you email address?

Comment (nwaniek) [reply docs/index.md]: You could change it to n@rochus.net, thanks!

**Review by nwaniek — APPROVED** [review]

File: docs/index.md (hunk 3 above)
Comment (MarcoTodescato) [reply docs/index.md]: great... I think we also need to update the NOTICE file stating your new affiliation if you are going to contribute in the future (which I hope :-D)

Comment (nwaniek) [reply docs/index.md]: Yes, I'll contribute soon-ish! The PR is almost ready, but I couldn't work and won't be able to work on it for some time: desktop PC is in a box waiting to be transported, and my notebook is unsuitable for developing and debugging blender extensions ;)

About my affiliation: You can either leave it off entirely, or simply add "Kavli Institute for Systems Neuroscience"

Comment (MarcoTodescato) [reply docs/index.md]: perfect! will do!

**Pull request merged**